docs/lang/articles/advanced/differentiable_programming.md

```diff
@@ -21,10 +27,10 @@ def compute_y():
     y[None] = ti.sin(x[None])
 ```
 
-Now if you want to get the derivative of y corresponding to x, i.e.,
-dy/dx. You may want to implement the derivative kernel by yourself:
+Now if you want to get the derivative of `y` corresponding to `x`:
```

Comment:
```suggestion
Now if you want to get the derivative of `y` with respect to `x`:
```
IIRC "with respect to" is the common mathematical language to describe differentiation.